.changeset/selfish-pans-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-neoboard-widget': minor
+---
+
+Show the collaborators cursor in presentation mode if edit slide active
```

Comment:
It sounds like cursors are shown during editing but this change adds a button that can be used to show the cursors.
```suggestion
Allows collaborators cursors to be shown in presentation mode if slide editing is enabled
```